src/main/java/com/github/awwkoala/selenium/page/FileUploadPage.java

```diff
@@ -27,10 +29,11 @@ public void goTo() {
     driver.get("http://theinternet.przyklady.javastart.pl/upload");
   }
 
-  public FileUploadPage chooseFileToUpload() {
+  public FileUploadPage chooseFileToUpload(String filename) {
+    PathUtils pathUtils = new PathUtils();
```

Comment:
I think in that situation you don't new to create an object of PathUtils every time. Moreover it can be static  method `getPathFromResources`

Reply:
.